modules/dnn/test/test_layers.cpp

```diff
@@ -65,13 +65,26 @@ static String _tf(TString filename)
 class Test_Caffe_layers : public DNNTestLayer
 {
 public:
+    void testLayerUsingOnnxModels(const String& basename, bool useCommonInputBlob = true, double l1 = 0.0, double lInf = 0.0,
```

Comment:
It's misleading change. Test naming and hierarchy defines that the test are Caffe tests. I proposed to extract base class and 2 derivatives: Test_Caffe_Layers and Test_ONNX_Layers or just have TestLayers as common class. @vpisarev What do you think?

Reply:
Onnx layers tests should be in `test_onnx_importer.cpp`. If we want to extract the tests for Caffe layer, I think we should move them to `test_onnx_importer.cpp`.

Reply:
Keep tests as simple as possible. There are no tests for tests so test code should avoid complications.
It makes sense to create new set of ONNX tests. At least we don't break already existed tests.